### PR TITLE
point demos to varnish, and fix imports to be defaulted

### PR DESCRIPTION
**Pull request by jonborchardt**

### CODE REVIEW

File: README-pt_BR.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ yarn add antd
 ## 🔨 Uso
 
 ```jsx
-import { Button, DatePicker } from 'antd';
+import { Button, DatePicker } from '@allenai/varnish';
```
Comment (codeviking) [inline README-pt_BR.md:86]: Woo!

File: README.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ yarn add @allenai/varnish
 Then use it like so:
 
 ```tsx
-import { Header, Button, ThemeProvider } from '@allenai/varnish'
+import { Header, Button, Theming } from '@allenai/varnish';
```
Comment (codeviking) [inline README.md:21]: Why `Themeing`? 

I'd think either `theme.Provider`, or `Theme.Provider`, but `Theming` is a little awkward.

File: components/header/index.tsx [hunk 3]
```diff
@@ -1 +1,12 @@
...
+export type HeaderProps = typeof OriginalHeader & {
+  HeaderColumns: typeof HeaderColumns;
+  HeaderTitle: typeof HeaderTitle;
+};
+
+const Header = OriginalHeader as HeaderProps;
+Header.HeaderColumns = HeaderColumns;
+Header.HeaderTitle = HeaderTitle;
+
+export default Header;
```
Comment (codeviking) [inline components/header/index.tsx:12]: What's all this doing? Maybe a comment or two to explain why we're doing this?

File: components/varnish/index.ts [hunk 4]
```diff
@@ -1,2 +1,7 @@
-export * from './Theme';
-export * from './ThemeProvider';
+import * as Theme from './Theme';
+import * as ThemeProvider from './ThemeProvider';
+
+export default {
+    ...Theme,
+    ...ThemeProvider,
+}
```
Comment (codeviking) [inline components/varnish/index.ts:7]: I wonder about the utility of maintaining the `components/varnish` package. To me I'm not sure if the distinction is useful, and particularly so for end users.  Does an end user care of it's part of varnish? Maybe not?

I'd probably just start plopping things into `components/`. That way there's no confusion about what varnish is, as, well, varnish is....everything!

It also prevents weird imports like `import xxxxx from '@allenai/varnish/lib/varnish'`, which looks a little silly.

File: report.20200522.123809.59197.0.001.json [hunk 5]
```diff
@@ -0,0 +1,901 @@
+
```
Comment (codeviking) [inline report.20200522.123809.59197.0.001.json:1]: What is this? Should we ignore / remove this?

File: README.md (hunk 2 above)
Comment (jonborchardt) [reply README.md:21]: theming includes an object called Theme
Theming.Theme
Theming.ThemeProvider

File: components/header/index.tsx (hunk 3 above)
Comment (jonborchardt) [reply components/header/index.tsx:12]: this puts us in line with a pattern used on all ant components.

File: components/varnish/index.ts (hunk 4 above)
Comment (jonborchardt) [reply components/varnish/index.ts:7]: we agree.
this will eventually go away.... we need it for v0

File: README.md (hunk 2 above)
Comment (codeviking) [reply README.md:21]: Yea, it just feels awkward as is.

What about `theme.Provider`?  Better yet, why add this separate package at all? I liked:

```
import { ThemeProvider } from '@allenai/varnish';`
```

Maybe less is more? I only think we need subpackages if there's a lot of related things that we want to keep grouped together.

File: report.20200522.123809.59197.0.001.json (hunk 5 above)
Comment (jonborchardt) [reply report.20200522.123809.59197.0.001.json:1]: yes

File: README.md (hunk 2 above)
Comment (jonborchardt) [reply README.md:21]: https://github.com/allenai/varnish/issues/86

Comment (codeviking) [reply README.md:21]: Sounds good 👍

File: components/header/index.tsx (hunk 3 above)
Comment (codeviking) [reply components/header/index.tsx:12]: Gotcha. Thanks.

**Comment by jonborchardt** [comment]
this was merged as part of a different pr

**Pull request closed without merging**